src/adhocracy_spd/adhocracy_spd/workflows/test_digital_leben.py

```diff
@@ -1,44 +1,17 @@
+"""
+Test the digital leben workflow, which is now in core named
+'debate'. Acts as a regression test.
+
+"""
 from pyramid import testing
 from pytest import fixture
 from pytest import mark
 from webtest import TestResponse
 
+from adhocracy_core.utils.testing import add_resources
```

Comment:
To make this work we need to change the wokflow to "debate" for this test  only.